### PR TITLE
removing histogram API support

### PR DESCRIPTION
**Pull request by shintasmith**
**WHAT**
This PR removes support for the API calls:
* /v2.0/{tenantId}/views/histograms/{metricName}
* /v1.0/{tenantId}/experimental/views/histograms/{metricName}

This is only removing the API. There are still a lot of histogram rollup related code that can be removed as well. But I will do that as a separate PR as it is more invasive.

**WHY**
Histogram is not being used. Product and development team decided to remove it

### CODE REVIEW

**Comment by shintasmith** [comment]
@usnavi @ChandraAddala @VinnyQ @izrik pls review

File: blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java [hunk 1]
```diff
@@ -151,20 +149,6 @@ public void executeRollupForLocatorTriggersExecutionOfRollupRunnable() {
...
-
-        // then
-        verify(rollupReadExecutor, times(1)).execute(Matchers.<HistogramRollupRunnable>any());
-        verifyNoMoreInteractions(rollupReadExecutor);
-        verify(executionContext, times(1)).incrementReadCounter();
-        verifyNoMoreInteractions(executionContext);
-        verifyZeroInteractions(rollupBatchWriter);
-    }
-
-    @Test
```
Comment (ChandraAddala) [inline blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java]: I see that you removed tests from ``LocatorFetchRunnable`` but I dont see any code being removed from ``LocatorFetchRunnable``.

**Comment by ChandraAddala** [comment]
Looks good.

File: blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java (hunk 1 above)
Comment (shintasmith) [inline blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java]: These 2 methods have been removed from ```LocatorFetchRunnable```:
* processHistogramForLocator
* executeHistogramRollupForLocator

Comment (ChandraAddala) [inline blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java]: I am blind.

Comment (shintasmith) [inline blueflood-core/src/test/java/com/rackspacecloud/blueflood/service/LocatorFetchRunnableTest.java]: Lots of red lines, I know :-)

**Comment by usnavi** [comment]
lgtm

**Comment by lakshmi-kannan** [comment]
>Histogram is not being used. Product and development team decided to remove it

It's rather sad. It could have been moved to a separate rollup module which could have enabled via Configuration.

**Pull request merged**